palindromic.py

```diff
@@ -1,3 +1,28 @@
-def get_longest_palindrome(s: str) -> str:
+'''
+Vodule
+'''
 
-    return "x"
+
+def is_palindrom(strng: str) -> bool:
+    '''
+
+    :param strng:
+    :return:
+    '''
+    return strng == strng[::-1]
+
+
+def get_longest_palindrome(strng: str) -> str:
+    '''
+
+    :param s:
+    :return:
+    '''
+    lngth = len(strng)
```

Comment:
What about palindromes in the middle of the string? I'm going to add test for this case